workflow/default_parameters/v24_2_2_4/defaults.yaml

```diff
@@ -0,0 +1,268 @@
+emod3d:
+  all_in_one: 1
+  bfilt: 4
+  bforce: 0
+  dampwidth: 0
+  dblcpl: 0
+  dmodfile: rho3dfile.d
+  dtts: 20
+  dump_itinc: 4000
+  dxout: 1
+  dxts: 5
+  dyout: 1
+  dyts: 5
+  dzout: 1
+  dzts: 1
+  elas_only: 0
+  enable_output_dump: 1
+  enable_restart: 1
+  ffault: 2
+  fhi: 0.0
+  fmax: 25.0
+  fmin: 0.01
+  freesurf: 1
+  geoproj: 1
+  intmem: 1
+  ix_ts: 99
+  ix_ys: 100
+  ix_zs: 100
+  iy_ts: 2
+  iy_xs: 60
+  iy_zs: 50
+  iz_ts: 1
+  iz_xs: 1
+  iz_ys: 1
+  lonlat_out: 1
+  maxmem: 1500
+  model_style: 1
+  nseis: 1
+  order: 4
+  pmodfile: vp3dfile.p
+  pointmt: 0
+  qbndmax: 100.0
+  qpfrac: 100.0
+  qpqs_factor: 2.0
+  qsfrac: 50.0
+  read_restart: 0
+  report: 100
+  restart_itinc: 20000
+  scale: 1
+  smodfile: vs3dfile.s
+  span: 1
+  stype: 2tri-p10-h20
+  swap_bytes: 0
+  ts_inc: 1
+  ts_start: 0
+  ts_total: 218
+  ts_xy: 1
+  ts_xz: 0
+  ts_yz: 0
+  tzero: 0.6
+  vmodel_swapb: 0
+  xseis: 0
+  yseis: 0
+  zseis: 0
+  pertbfile: test
+hf:
+  nbu: 4
+  ift: 0
+  flo: 0.02
+  fhi: 19.9
+  nl_skip: -99
+  vp_sig: 0.0
+  vsh_sig: 0.0
+  rho_sig: 0.0
+  qs_sig: 0.0
+  ic_flag: true
+  velocity_name: "-1"
+  dt: 0.02
+  t_sec: 0.0
+  sdrop: 50.0
+  rayset: [1, 2]
+  no_siteamp: false
+  fmax: 10.0
+  kappa: 0.045
+  qfexp: 0.6
+  rvfac: 0.8
+  rvfac_shal: 0.7
+  rvfac_deep: 0.7
+  czero: 2.1
+  calpha: -99.0
+  mom: null
+  rupv: null
+  site_specific: false
+  vs_moho: 999.9
+  fa_sig1: 0.0
+  fa_sig2: 0.0
+  rv_sig1: 0.1
+  path_dur: 1
+  dpath_pert: 0.0
+  stress_parameter_adjustment_tect_type: 1
+  stress_parameter_adjustment_target_magnitude: null
+  stress_parameter_adjustment_fault_area: null
+  stoch_dx: 2.0
+  stoch_dy: 2.0
+  seed: 0
+srf:
+  resolution: 0.4
+  genslip_seed: 1
+  genslip_dt: 0.05
+  srfgen_seed: 1
+  genslip_version: 5.4.2
+velocity_model:
+  min_vs: 0.5
+  resolution: 0.4
+  ds_multiplier: 1.2
+  dt: 0.02
+  version: "2.06"
+  topo_type: "SQUASHED_TAPERED"
+  vs30: 500.0
+  s_wave_velocity: 3500.0
+  pgv_interpolants: [[3.5, 0.015], [4.1, 0.0375], [4.7, 0.075], [5.2, 0.15], [5.5, 0.25], [5.8, 0.4], [6.2, 0.7], [6.5, 1.0], [6.8, 1.35], [7.0, 1.65], [7.4, 2.1], [7.7, 2.5], [8.0, 3.0]]
+bb:
+  flo: 0.5
+  dt: 0.02
+  fmidbot: 0.5
+  fmin: 0.25
```

Comment:
These seem different to what I saw in root_defaults.yaml 20.4.1.4 fmin: 0.2 and fmidbot: 0.25, as well as dt: 0.005. Do you know why these would be different? Not gonna compare everything but these stuck out to me

Reply:
That `dt` value is incorrect for the old root_defaults. The value `dt / 20` is what is used for EMOD3D to ensure stability, that value seems to be the `dt` value used for 100m simulations instead of 400m simulations. As for `fmin` and `fmidbot` I have no idea, but I would guess that maybe the root defaults is not a good place to look if it may have been copied from the 100m defaults.

Reply:
There are a few hints in the codebase that `dx/20` is implied: 

1. It is consistent with the values for 200m and 100m that I copied from the root defaults of the old simulation workflow
2. Places like https://github.com/ucgmsim/Pre-processing/blob/a512a1866f3edd6425a2b68d9cf96ce5a39f351c/SrfGen/slurm/gcmt/GCMT2SRF_VM.sl#L43 where we generate SRFs for 400m simulations with `dt = 0.4 / 20 = 0.02`